src/HotChocolate/Core/test/Execution.Tests/DependencyInjection/RequestExecutorBuilderExtensions_IdSerializer.Tests.cs

```diff
@@ -275,7 +275,7 @@ public string Serialize<T>(NameString schemaName, NameString typeName, T id)
                 return "mock";
             }
 
-            public IdValue Deserialize(string serializedId)
+            public IdValue Deserialize(string serializedId, Type resultType = null)
```

Comment:
This demonstrates that it's a breaking change but netstandard2.0 doesn't have default interface implementations so I guess this means a v12? Or a v11.1 with some clear release notes.

Reply:
Outdated as now with a default implementation this isn't needed :) The system works!